zip-codes/scripts/search.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
 (function(module) {
-
+  webDB.execute(
+    'SELECT DISTINCT state FROM zips ' +
+    'ORDER BY state DESC;',
+    function(rows){
+      console.log(rows);
+      if (rows.length){
+        rows.forEach(function(a){
+          $('#state-select').append('<option>' + a.state + '</option>');
+        });
+      }
+      $('#state-select').on('change', function() {
+        $('#city-select').find('option').remove().end();
+        console.log($(this).val(), 'state value');
+        if ($(this).val()) {
+          webDB.execute(
+            'SELECT DISTINCT city FROM zips WHERE state="' + $(this).val() + '"' +
+            ' ORDER BY city DESC;',
```

Comment:
Excellent work populating your city select box! You have the same backwards-ordering happening here, though. You could instead ORDER BY city ASC; :)